src/openrct2/drawing/Drawing.Sprite.cpp

```diff
@@ -180,6 +180,7 @@ void mask_scalar(
 static rct_gx _g1 = {};
 static rct_gx _g2 = {};
 static rct_gx _csg = {};
+static rct_g1_element _scrollingText[256]{};
```

Comment:
Shouldn't this be
```suggestion
static rct_g1_element _scrollingText[MAX_SCROLLING_TEXT_ENTRIES]{};
```
?

Reply:
That constant is defined in ScrollingText.cpp

Reply:
That's no good reason to hardcode it here. It completely defeats the point of having a constant in the first place.

Reply:
Where do you suggest I define the constant? It would be required in sprites.h and ScrollingText.cpp

Reply:
I suggest we ask Ted or Duncan. I would like one of them to review the PR anyway.

Reply:
I've introduced a header file called GameLimits.h, been wanting to do that for a while now. I think this simplifies such cases and makes it easier to control the compilation out of one place.